routes/customer/order.py

```diff
@@ -0,0 +1,61 @@
+from flask import render_template, request
+from . import customer_order_routes
+from routes import connect_to_database
+import database.db_connector as db
+
+@customer_order_routes.route('/add', methods = ['GET', 'POST'])
+@connect_to_database
+def customer_order_add(db_connection):
+    if request.method == 'POST':
+        cid = request.form['customerid']
+        street = request.form['street']
+        city = request.form['city']
+        zip_code = request.form['zip']
+        state = request.form['state']
+        data = [int(cid), street, city, zip_code, state]
+        query = 'INSERT INTO Orders (customerId, street, city, zip, state) VALUES (%s, %s, %s, %s, %s);'
+        cursor = db.execute_query(db_connection=db_connection, query=query, query_params=data)
+
+    query = 'SELECT id, firstName, lastName FROM Customers;'
+    cursor = db.execute_query(db_connection=db_connection, query=query)
+    customers = cursor.fetchall()
+    return render_template('customer/order/add-order.j2', customers_list = customers)
+
+@customer_order_routes.route('/assign-item', methods = ['GET', 'POST'])
+@connect_to_database
+def customer_order_assign_item(db_connection):
+    if request.method == 'POST':
+        oid = request.form['orderid']
+        item_id = request.form['itemid']
+        quantity = request.form['quantity']
+        data = [int(oid), int(item_id), int(quantity)]
+        query = 'INSERT INTO OrderItems (orderId, itemId, quantity) VALUES (%s, %s, %s);'
```

Comment:
Notice these parameters are now `%s` instead of `%d`.  Again, the database engine did not like arguments such as this to be anything other than a string.  I suppose it makes sense.  SQL queries are strings.